src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
```

Comment:
The `<html>` tag should not be self-closing. Remove the `/` at the end of the tag to correct this.